skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillDTO.java

```diff
@@ -0,0 +1,28 @@
+package com.RDS.skilltree.Skill;
+
+import com.RDS.skilltree.User.UserModel;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SkillDTO {
+    private UUID id;
+    private SkillType type;
+    private String name;
+    private Set<UserModel> users;
+
+    public static SkillDTO toDto(SkillModel skillModel) {
+        return SkillDTO.builder()
+                .id(skillModel.getId())
+                .name(skillModel.getName())
+                .type(skillModel.getType())
+                .users(skillModel.getUsers())
```

Comment:
This is returning the users model without any filters. Since users model might contain sensitive info, we should covert it to DTO with minimum fields required for the feature. 

```suggestion
                .users(skillModel.getUsers().toDTO());
```

Reply:
This changes I did already, had to discuss few things with Tejas, will push these changes in the evening. Thanks for pointing out.

Reply:
pushed these changes.